tfhe/src/integer/gpu/server_key/radix/tests.rs

```diff
@@ -916,6 +920,100 @@ where
     }
 }
 
+fn integer_unchecked_scalar_eq<P>(param: P)
+where
+    P: Into<PBSParameters> + Copy,
+{
+    let gpu_index = 0;
+    let device = CudaDevice::new(gpu_index);
+    let stream = CudaStream::new_unchecked(device);
+
+    // let (_, h_sks) = KEY_CACHE.get_from_params(param, IntegerKeyKind::Radix);
```

Comment:
This line can be removed.

Reply:
I will remove all occurrences in this file.